starter-web-netty/src/test/groovy/io/micronaut/starter/netty/ReportAnalyticsSpec.groovy

```diff
@@ -29,7 +30,7 @@ import java.util.concurrent.CompletableFuture
         value =  "http://localhost:8080/analytics/report")
 @Property(
         name = "micronaut.http.services.analytics.token",
-        value =  "false")
+        value =  "test-api-key")
```

Comment:
can you add a non happy path test?

Reply:
There are two non-happy-path tests already:
- [`test save generation data without api key`](https://github.com/micronaut-projects/micronaut-starter/pull/1980/files#diff-a127201424deff00e0a159547527155a6b80525c70ffdf3d02651d947330ca51R63)
- [`test save generation data with a wrong api key`](https://github.com/micronaut-projects/micronaut-starter/pull/1980/files#diff-a127201424deff00e0a159547527155a6b80525c70ffdf3d02651d947330ca51R82)

What other scenarios do you want to test, specifically?

Reply:
I would like a a test which verifies how starter-web-netty behaves if the anlaytics server returns a 401?

Reply:
the test you linked test analytics. I want tests for starter-web-netty

Reply:
Fair enough. On it 👍